.github/workflows/external.yml

```diff
@@ -15,12 +15,18 @@ jobs:
     - name: Build
       run: make REDIS_CFLAGS=-Werror
     - name: Start redis-server
-      run: ./src/redis-server --daemonize yes
+      run: ./src/redis-server --daemonize yes --logfile external-redis.log
     - name: Run external test
       run: |
           ./runtest \
             --host 127.0.0.1 --port 6379 \
             --tags -slow
+    - name: Archive redis log
+      if: ${{ failure() }}
+      uses: actions/upload-artifact@v2
+      with:
+        name: test-external-redis-log
+        path: external-redis.log
```

Comment:
I think i did this here originally in order to understand something and then cherry picked to a separate PR. There's no conflict because the change is identical.
What do you suggest I do?

Reply:
looking at the branch of this PR, i see this block only once, and since there's no conflict, i suppose that we might get this block twice when we merge it.
anyway, you have two options:
1. delete this code from your branch
2. rebase (merge unstable into your branch), and make sure the block only appears twice.
either way, just make sure the diff between your branch and the `git merge-base` sha, shows no changes to this file (you can use the diff tool to revert them).